src/resources/Validator.js

```diff
@@ -141,6 +141,12 @@ Validator.prototype.validate = function(submission, next) {
   var error = null;
   debug('Starting validation');
 
+  // Skip validation if no data is provided.
+  if (!submission.data) {
```

Comment:
Is this dangerous? If there is no data we should still fail if they are trying to do a POST or PUT